### ISSUE TITLE
Invalid requestPolicy passed to useQueryJs

### ISSUE THREAD

**Issue by baransu** [FormidableLabs/reason-urql#137]
```reason
[@bs.deriving abstract]
type useQueryArgs = {
  query: string,
  variables: Js.Json.t,
  [@bs.optional]
  requestPolicy: UrqlTypes.requestPolicy,
  [@bs.optional]
  pause: bool,
};
```
This type doesn't convert nested `UrqlTypes.requestPolicy` type when creating. 

When doing:
```reason
let args =
    useQueryArgs(
      ~query=request##query,
      ~variables=request##variables,
      ~requestPolicy?,
      ~pause?,
      (),
    );
```
is should be 
```reason
let args =
    useQueryArgs(
      ~query=request##query,
      ~variables=request##variables,
      ~requestPolicy=requestPolicy->Belt.Option.map(UrqlTypes.requestPolicyToJs),
      ~pause?,
      (),
    );
```
This way string instead of int will be passed int `"requestPolicy"` key in `urql` options.

**Comment by parkerziegler** [comment]
Aha, right you are @baransu! Good catch, and thanks for watching out. Do you want to PR the fix? I'm also happy to do it and co-author you on the commit 😄 Thanks as well for all the hard work on `graphql_ppx_re`, really appreciate that!

**Comment by baransu** [comment]
I won’t have time this week to do that. If you want feel free to fix it 😉

Thanks for using `graphql_ppx_re` ❤️

**Comment by parkerziegler** [comment]
Released in [v1.4.1](https://github.com/FormidableLabs/reason-urql/releases/tag/v1.4.1).
